pkg/controller/unifiedpushserver/unifiedpushserver_controller.go

```diff
@@ -939,6 +978,50 @@ func (r *ReconcileUnifiedPushServer) Reconcile(request reconcile.Request) (recon
 	}
 	//#endregion
 
+	//#region Monitoring
+	//## region ServiceMonitor
+	serviceMonitor := &monitoringv1.ServiceMonitor{ObjectMeta: metav1.ObjectMeta{Name: "unifiedpush", Namespace: instance.Namespace}}
```

Comment:
```suggestion
	serviceMonitor := &monitoringv1.ServiceMonitor{ObjectMeta: metav1.ObjectMeta{Name: instance.Name, Namespace: instance.Namespace}}
```

I think using the name of the CR would be a little better for all of these resources, wdyt?

Reply:
Yes, I'm happy to that. The reason for using this change here was the intention that there would be no change to the resources made during this change (to limit upgrade considerations), only that the operator would create it rather than from a file. Seeing as the Name value doesn't get update in the reconcile function I can add some logic in there to remove it if it has the old name.